mvn clean install fails

Hi guys,

currently I am working on a PR for my other issue (#64). After setting up the repository locally, I wanted to build it first. But `mvn clean install` fails, with the error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.5:sign (sign-artifacts) on project metrics-datadog-parent: Exit code: 2 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

After checking the configuration, I saw that the `maven-gpg-plugin` is configured to run in the `verify`-phase. According to reference of the [maven lifecycle](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference), I expect this step during the deploy phase.

--- a/pom.xml
+++ b/pom.xml
@@ -121,7 +121,7 @@
                 <executions>
                     <execution>
                         <id>sign-artifacts</id>
-                        <phase>verify</phase>
+                        <phase>deploy</phase>
                         <goals>
                             <goal>sign</goal>
                         </goals>

